scripts/clean_results_csv.py

```diff
@@ -1,3 +1,10 @@
+"""
+This script cleans up the CSV currently (as of 2022.10.14) generated by `hallucinate_fragalysis.py` into
+a format the following scripts can use.
+
+Example Usage:
+    python clean_results_csv.py -i /lila/data/chodera/kaminowb/stereochemistry_pred/mers/mers_fragalysis/posit_hybrid_no_relax_keep_water_filter/all_results.csv -o /lila/data/chodera/paynea/posit_hybrid_no_relax_keep_water_filter_frag -d -s
```

Comment:
best to format this in the same way as the docstring in scripts/fauxalysis_from_docking.py (i.e. with newlines)